test/filesystem.test

```diff
@@ -35,12 +35,12 @@ test_expect_success mkfs_f2fs,sload_f2fs,fsck_f2fs "f2fs" "
 	fsck.f2fs images/test.f2fs
 "
 
+exec_test_set_prereq btrfs
 exec_test_set_prereq mkfs.btrfs
-exec_test_set_prereq sload.btrfs
-exec_test_set_prereq fsck.btrfs
-test_expect_success mkfs_btrfs,fsck_btrfs "btrfs" "
+test_expect_success mkfs_btrfs,btrfs "btrfs" "
```

Comment:
For some reason the build tests are stuck in a waiting state. But I'm pretty sure, that you need to add `btrfs-progs` to the `apt-get install ...` line in `.github/workflows/tests.yml`. Otherwise, this test is skipped in the CI.

Reply:
Good point, I've added that.

I think you might have to approve the workflow run because this is my first contribution to genimage, at least I see "This workflow requires approval from a maintainer" in the status box.

Reply:
I'm aware of the approval process. I've had to do that for other PRs in the past, but there was no button for it before. It was there now, so I could trigger it. Maybe the update knocked something loose...

Reply:
Huh, that's odd. Anyway, thanks for the review and merge! :slightly_smiling_face: